packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/package.json.ts

```diff
@@ -0,0 +1,52 @@
+import { TemplateFn } from '@cli/types';
+import { sharedPkgJsonField } from '@partials/pkg-shared/sharedPkgJsonFieldsT';
+
+const json: TemplateFn = ({ data, utils }) => /* JSON */ `{
+	${sharedPkgJsonField(utils.pkgName(data.appName))}
+	"scripts": {
+		"dev": "npm run build-scripts && next dev",
+		"build": "npm run build-scripts && next build && cp -r .next/static .next/standalone/.next && cp -r public .next/standalone",
+		"build-scripts": "tsc ./scripts/*.ts",
+		"start": "node .next/standalone/server.js",
+		"build:mono": "next build",
+		"start:mono": "next start",
+		"lint": "next lint",
+		"lint:fix": "next lint --fix && npm run prettier:fix",
+		"prettier": "prettier '**/*.{js,jsx,md}' --check --ignore-path .prettierignore",
+		"prettier:fix": "prettier '**/*.{js,jsx,,md}' --write --ignore-path .prettierignore",
+		"test": "vitest run",
+		"update-snapshots": "vitest run --update --silent",
+		"coverage": "vitest run --coverage"
+	},
+	"dependencies": {
+		"@pantheon-systems/drupal-kit": "${data.drupalKitVersion}",
+		"@pantheon-systems/nextjs-kit":  "${data.nextjsKitVersion}",
+		"dotenv": "^16.0.2",
+		"@formatjs/intl-localematcher": "^0.4.0",
```

Comment:
Small thing, move this to the top of `dependencies` – ideally these should be in alphabetical order.